SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -660,6 +661,22 @@ ERPCResult USpatialSender::SendRPCInternal(UObject* TargetObject, UFunction* Fun
 
 		Worker_CommandRequest CommandRequest = CreateRPCCommandRequest(TargetObject, Payload, ComponentId, RPCInfo.Index, EntityId);
 
+		// This check happens after building the command request as building the request also sets the entity ID.
+		if (ComponentId == SpatialConstants::SERVER_TO_SERVER_ENDPOINT_COMPONENT_ID_LEGACY)
+		{
+			ServerToServerRPCEndpointLegacy* CrossServerRPCComponent = StaticComponentView->GetComponentData<ServerToServerRPCEndpointLegacy>(EntityId);
+			if (CrossServerRPCComponent == nullptr)
+			{
+				// We haven't seen the component yet to send the RPC to.
+				return ERPCResult::NoActorChannel;
+			}
+			else if (!CrossServerRPCComponent->bReady)
```

Comment:
As said before - I don't think we can make this a meaningful concept for cross server.
I think the way to go with this is to make it redundant, like we did with ring buffer client-server rpcs.

Reply:
removed